app/views/like/_like.html.erb

```diff
@@ -12,11 +12,11 @@
     <% if node.type == 'page' %>
       <% if node.has_tag('locked') && (current_user ? current_user.basic_user? : true) %>
         <a href="/wiki/locked" class='btn btn-outline-secondary btn-circle btn-sm' rel="tooltip" title="This page is locked. Click to learn why." data-placement="top">
-          <i class="ml-1 ff fa fa-lock"></i>
+          <i class="fa fa-lock"></i>
         </a>
       <% elsif current_user && current_user.first_time_poster %>
         <a class='btn btn-outline-secondary btn-circle btn-sm' rel="tooltip" title="You can create the wiki once your research note/question is approved by moderators. Click to learn why." data-placement="top" href="https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters">
-          <i class="ml-1 ff fa fa-lock"></i>
+          <i class="fa fa-lock"></i>
```

Comment:
Hi @Sushil333, these changes aren't required in this PR. Thanks! 

Can you also remove the `package.json` and `yarn.lock` files? Thanks!

Reply:
should I delete package.json & yarn.lock?

Reply:
Hi @Sushil333, the files are still there. You also don’t need to delete the files, just copy and paste the original into your local version of the file.